config.js

```diff
@@ -24,51 +24,66 @@
 var config = {};
 
 config.lwm2m = {
-    logLevel: 'DEBUG',
-    port: 60001,
-    defaultType: 'Device',
-    ipProtocol: 'udp4',
-    serverProtocol: 'udp4',
-    formats: [
-        {
-            name: 'application-vnd-oma-lwm2m/text',
-            value: 1541
-        },
-        {
-            name: 'application-vnd-oma-lwm2m/tlv',
-            value: 1542
-        },
-        {
-            name: 'application-vnd-oma-lwm2m/json',
-            value: 1543
-        },
-        {
-            name: 'application-vnd-oma-lwm2m/opaque',
-            value: 1544
-        }
-    ],
-    writeFormat: 'application-vnd-oma-lwm2m/text',
-    types: [ ]
+  logLevel: 'DEBUG',
```

Comment:
It seems to me you have automaticaly reformat everything to two spaces per tab. Please, respect the original four spaces, as it also will make the PR easier to read.